node/node.go

```diff
@@ -227,10 +225,8 @@ func NewNode(
 		return nil, fmt.Errorf("failed to create new store: %w", err)
 	}
 
-	socketsFilterer, err := indexer.NewOperatorSocketsFilterer(serviceManagerAddress, client)
-	if err != nil {
-		return nil, fmt.Errorf("failed to create new operator sockets filterer: %w", err)
-	}
+	// TODO: OperatorSocketsFilterer functionality was removed due to indexer deprecation.
```

Comment:
Interesting. So this seems to be the only production use case of the inmem indexer right now? Graph node would not be a good option because that would require every operator to take a dependency on that infrastructure.